cloud_governance/common/clouds/aws/cloudtrail/cloudtrail_operations.py

```diff
@@ -157,8 +158,9 @@ def get_username_by_instance_id_and_time(self, start_time: datetime, resource_id
         @param resource_type:
         @return: if user not found it return empty string
         """
-        if self.__get_time_difference(start_time=start_time) <= self.SLEEP_SECONDS:
-            time.sleep(self.SLEEP_SECONDS)
+        delay_seconds = int(os.environ.get('SLEEP_SECONDS', self.SLEEP_SECONDS))
+        if self.__get_time_difference(start_time=start_time) <= delay_seconds:
+            time.sleep(delay_seconds)
```

Comment:
Delay_seconds should be upper

Reply:
it's not global, local variable.
I put it because we are testing tagging.